src/main.rs

```diff
@@ -41,11 +41,27 @@ fn parse_speed(arg: &str) -> Result<u8, String> {
     Ok(s)
 }
 
+fn parse_color_space(s: &str) -> Result<ColorSpace, String> {
+    match s.to_lowercase().as_str() {
+        "srgb" => Ok(ColorSpace::Srgb),
+        "displayp3" => Ok(ColorSpace::DisplayP3),
+        "rec2020Pq" => Ok(ColorSpace::Rec2020Pq),
```

Comment:
Otherwise `to_lowercase()` can't ever match:

```suggestion
        "rec2020pq" => Ok(ColorSpace::Rec2020Pq),
```